Unix/http/httpauth.c

```diff
@@ -1513,30 +1532,40 @@ static unsigned char *_BuildAuthResponse(_In_ const char *pProtocol,
 
     const char *header_start = NULL;
     int header_start_len = 0;
+    int header_tail_len  = 0;
 
     struct _EncodeContext encode_context = { 0 };
 
     switch (httpStatus)
     {
+    case 0: // We will add the header info later. Just build a WWW-Authenticate: 
+        header_start = HEADER_START_AUTH_ONLY;
+        header_start_len = strlen(HEADER_START_AUTH_ONLY);
```

Comment:
don't strlen constant strings! Note, there are other places this is done so fix them also!